internal/web_package/assembler.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2018 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+function main(args) {
+  const params = fs.readFileSync(args[0], {encoding: 'utf-8'}).split('\n').filter(l => !!l);
+
+  const outdir = params.shift();
+  if (!fs.existsSync(outdir)) fs.mkdirSync(outdir);
+
+  for (const f of params) {
```

Comment:
We'll ignore the nested directory structure for now?

Reply:
Maybe use a list files function to get the file list and then just iterator over that. Here's an example: https://github.com/bazelbuild/rules_nodejs/blob/master/internal/npm_install/generate_build_file.js#L165

Reply:
Sounds good

Reply:
problem with the nested directory structure is that we don't know what is the right base path to strip

we probably need to match ts_devserver with a list of strip_prefix rules
will punt this to the next story... don't have enough time to do that now.